src/dsp/pdr.c

```diff
@@ -976,9 +979,9 @@ int pldm_entity_association_pdr_add_check(pldm_entity_association_tree *tree,
 	if (!tree || !repo) {
 		return 0;
 	}
-
+	uint32_t record_handle = 0;
```

Comment:
what happens when we send a record_handle from caller API? it will be set to 0?

Reply:
Function `pldm_entity_association_pdr_add_check` does not take record handle as an argument. So user can not send record_handle to this function and record handle is defaulted to 0 inside the function.

Reply:
yes, so that record_handle variable is redundant right?, we can directly pass 0 as how it was earlier?

Reply:
Earlier `entity_association_pdr_add` was taking "uint32_t record_handle" as argument now I have changed it to take pointer to record handle "uint32_t *record_handle". So we are expected to pass the address of the variable and not the value. To get the address of variable, created a variable record_handle and passing address of this variable to function `entity_association_pdr_add`.

Reply:
acknowledged